pages/customizing-bootstrap-4.md

```diff
@@ -27,4 +27,8 @@ For example to use the border-radius mixin:
 ```
 Make sure you import only partials and not main sass files, otherwise you will end up generating duplicate css which might cause issues
 
+To change the default bootstrap settings like colors, border-radius, etc, add or change the value of the property in the partial file `src/main/webapp/content/scss/_bootstrap-variable.scss`
```

Comment:
I probably missed punctuation. This file was added to master so wont be available in older releases